src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/config/InterceptorConfig.java

```diff
@@ -24,5 +29,7 @@ public InterceptorConfig(LoggingInterceptor loggingInterceptor) {
     @Override
     public void addInterceptors(@NonNull InterceptorRegistry registry) {
         registry.addInterceptor(loggingInterceptor);
+        registry.addInterceptor(new TokenPermissionsInterceptor());
```

Comment:
for the token and user auth interceptors, we don't want them to run against all endpoints which is how they are configured here. On Roe we grouped them by User and Internal endpoints. We have the  API endpoints documented by the architect so we can configure what auth runs against what endpoint here.

Reply:
I have replicated what we have in roe it does not use groups but require specific individual endpoint types. This looks like something we'll need to build up as we add to this rather than try and add in one go.